google/cloud/datastore/aggregation.py

```diff
@@ -327,8 +407,7 @@ def _build_protobuf(self):
         """
         pb = self._aggregation_query._to_pb()
         if self._limit is not None and self._limit > 0:
-            for aggregation in pb.aggregations:
-                aggregation.count.up_to = self._limit
+            pb.nested_query.limit = self._limit
```

Comment:
can we add some more limit integration tests on SUM/AVG to confirm this is working as expected?

Reply:
Good point, added. Let me know if any other gaps stand out to you